spyder/plugins/editor/widgets/editor.py

```diff
@@ -2410,11 +2411,11 @@ def __check_file_status(self, index):
                         self.msgbox = QMessageBox(
                             QMessageBox.Question,
                             self.title,
-                            _("The file <b>{}</b> has been modified outside "
-                              "Spyder."
+                            _("It looks like <b>{}</b> has been modified "
+                              "outside Spyder. The working copy is from {} milliseconds ago."
                               "<br><br>"
                               "Do you want to reload it and lose all your "
-                              "changes?").format(name),
+                              "changes?").format(name, dt),
```

Comment:
We're no longer able to change translation strings in Spyder 5, so please revert these changes. We'll make sure to apply them for Spyder 6 when merging your pull request with `master`.